tools/cellprofiler/common.xml

```diff
@@ -0,0 +1,445 @@
+<tool id="cp_common" name="Cellprofiler common start modules" version="@CP_VERSION@">
+    <description>
+        The first 4 modules in Cellprofiler to start a pipeline, namely Images,Metadata, NameAndTypes, Groups 

```

Comment:
@beatrizserrano should review all these names + descriptions ... they need to be meningful and short. Both make up the name in the tool-panel.

Reply:
I think that the names should stay the same, as this is how they are shown in CellProfiler and the documentation. I've taken the descriptions from the documentation and respected mostly all of them, some were quite long and I summarised a bit :)

Reply:
Yes they have weird names..I agree we should keep the name same as how they appear in CP. Thank for providing the description